app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -34,7 +34,7 @@
 import com.firebase.ui.auth.AuthUI;
 import com.firebase.ui.auth.AuthUI.IdpConfig;
 import com.firebase.ui.auth.IdpResponse;
-import com.firebase.ui.auth.ui.ResultCodes;
+import com.firebase.ui.auth.ResultCodes;
```

Comment:
The library version should probably be bumped to `1.1.0` since this is slight breakage.